### DISCUSSION TITLE
After disabling the authentication i am getting the login page but where can i check/configure for username password , for reference find below screenshot

### DISCUSSION THREAD

**Discussion by balajin2022** [rackslab/Slurm-web#248] [Q&A]
![Screenshot from 2024-01-22 20-00-17](https://github.com/rackslab/slurm-web/assets/103120252/86c90023-5de3-4fa1-9a2c-cd3f712267d1)

**Comment by rezib** [comment]
Hi @balajin2022, did you read https://github.com/rackslab/slurm-web/issues/243#issuecomment-1900666260?

**Comment by rezib** [comment]
This question was about deprecated Slurm-web 2, then I close the discussion.
